docs/DeveloperGuide.md

```diff
@@ -409,24 +401,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+| `* * *`  | student                        	        | add tasks for each module tag                    | remind myself of all the tasks that are under those tags.              |
+| `* * *`  | student                        	        | delete tasks for each module tag                 | remove the tasks that are no longer required for that module           |
+| `* * *`  | student with many project groups	        | add the meeting platform links to each module tag| conveniently contact the team or initiate a team meeting               |
+| `* * *`  | student with many project groups	        | delete the meeting platform link for each tag    | remove the link if it has been added wrongly                           |
+| `* *`    | student                        	        | clear all completed tasks under each tag         | remove the tasks that are no longer required                           |
 | `* *`    | user                                       | update a tag (name)                              | correct the misspelled tag name                                        |
 | `* *`    | user                                       | sort the tags by its tag name                    | view tags in alphabetical order                                        |
 | `* *`    | user with many contacts                    | sort the contacts by name                        | view contacts in alphabetical order                                    |
-| `* *`    | student                        	        | add tasks for each module tag                    | remind myself of all the tasks that are under those tags.              |
-| `* *`    | student                        	        | delete tasks for each module tag                 | remove the tasks that are no longer required for that module           |
 | `* *`    | student                        	        | mark the status of a particular task as done     | keep track of the different task progress under that tag.              |
-| `* *`    | student with many project groups	        | add the meeting platform links to each module tag| conveniently contact the team or initiate a team meeting               |
-| `* *`    | student with many project groups	        | delete the meeting platform link for each tag    | remove the link if it has been added wrongly                           |
 | `* *`    | student with many project groups	        | add remarks on the module tag                    | find module-related information from the module tag                    |
 | `* *`    | student with many project groups	        | delete the comments on the module tag            |                                                                        |
 | `*`      | long-time user	                            | archive old module tags 	                       | keep my contacts up to date                                            |
```

Comment:
should we remove user stories like this?

Reply:
I think since one star means unlikely to have it is alright for them to be still there?

Reply:
ah okie!